contents/blog/posthog-vs-statsig.mdx

```diff
@@ -0,0 +1,294 @@
+
+This post compares their platforms, experimentation features, pricing, and more.
+
+## How is PostHog different?
+
+### 1. It is an all-in-one platform
+
+PostHog brings together all the tools engineers need for testing, releasing, and measuring the success of new features. Feature flags and A/B testing are only part of a suite of tools PostHog offers.
+
+![PostHog homepage features](../images/blog/posthog-vs-statsig/all.png)
+
+PostHog combines usage, performance, and behavioral data with flags and experiments. PostHog’s [data warehouse](/docs/data-warehouse) also enables you to pull in data from external sources.
+
+Having all these product and data tools together enables you to do better analysis of shipped features and make better decisions about what you are building next. 
+
+### 2. PostHog is built for engineers
+
+Anyone can use PostHog – just [ask our marketing team](/blog/posthog-marketing) – but PostHog is designed to meet the needs of engineers. Our goal is to help developers [be better at product](/blog/helping-engineers-to-product).
+
+### 3. We’re transparent (in many ways)
+
+PostHog is open source. Our code, culture, and strategy are public [on GitHub](https://github.com/PostHog/posthog) and our [handbook](/handbook). While Statsig open sourced some code (like SDKs), most is closed source.
+
+We're also self-serve. No need to "talk to sales" or "schedule a demo" unless you want to – we're always [happy to chat](/book-a-demo). You can sign up and try all the features PostHog offers for free.
+
+## Comparing PostHog and LaunchDarkly
+
+### Platform
+
+<ComparisonTable column1="PostHog" column2="Statsig">
+  <ComparisonRow column1={true} column2={true} feature="A/B testing" description="Test changes and analyze impact" />
+  <ComparisonRow column1={true} column2={true} feature="Feature flags" description="Roll out features safely; toggle features for cohorts or individuals" />
+  <ComparisonRow column1={true} column2={true} feature="Product analytics" description="Track events and conversion; analyze user behavior" />
+  <ComparisonRow column1={false} column2={true} feature="Configurator" description="Replace hard-coded values in your app with config values" />
+  <ComparisonRow column1={true} column2={false} feature="Session replays" description="Watch real users use your product; debug behavior" />
+  <ComparisonRow column1={true} column2={false} feature="User surveys" description="Ask users for qualitative feedback and gather responses" />
+  <ComparisonRow column1={true} column2={false} feature="Open source" description="Code publicly accessible" />
+  <ComparisonRow column1="In beta" column2={false} feature="Data warehouse storage" description="Store large amounts of analytics data" />
+  <ComparisonRow column1={false} column2={true} feature="Data warehouse experiments" description="Run A/B tests natively in your existing warehouse" />
```

Comment:
Yeah I would say so! Maybe a NIT is `running A/B tests on the data within your existing data warehouse`. The current phrasing tripped me out for a second because I just wasn't sure what running an A/B test in a warehouse meant